From 6d0cc6dbc4b503702ce4064521b87e1c1fae888a Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Wed, 19 Aug 2015 23:34:09 +0900
Subject: [PATCH] Add the MPLs to /usr/share/common-licenses.

Closes: #768292
Wording: Charles Plessy <plessy@debian.org>
Seconded: Russ Allbery <rra@debian.org>
Seconded: Andrey Rahmatullin <wrar@debian.org>
---
 debian/changelog         |  5 +++++
 policy.sgml              | 13 ++++++++-----
 upgrading-checklist.sgml |  6 +++++-
 3 files changed, 18 insertions(+), 6 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index d48eac8..ad66bd1 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -31,6 +31,11 @@ debian-policy (3.9.7.0) unstable; urgency=low
     Seconded: Bill Allombert <ballombe@debian.org>
     Seconded: Charles Plessy <plessy@debian.org>
     Closes: #106073
+  * Policy: Add the MPLs to /usr/share/common-licenses.
+    Wording: Charles Plessy <plessy@debian.org>
+    Seconded: Russ Allbery <rra@debian.org>
+    Seconded: Andrey Rahmatullin <wrar@debian.org>
+    Closes: #768292
 
  -- Bill Allombert <ballombe@debian.org>  Fri, 08 May 2015 15:10:02 +0200
 
diff --git a/policy.sgml b/policy.sgml
index 404dc73..36bed13 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -10820,9 +10820,10 @@ END-INFO-DIR-ENTRY
 	<p>
 	  Packages distributed under the Apache license (version 2.0), the
 	  Artistic license, the GNU GPL (versions 1, 2, or 3), the GNU
-	  LGPL (versions 2, 2.1, or 3), and the GNU FDL (versions 1.2 or
-	  1.3) should refer to the corresponding files
-	  under <file>/usr/share/common-licenses</file>,<footnote>
+	  LGPL (versions 2, 2.1, or 3), the GNU FDL (versions 1.2 or 1.3),
+	  and the Mozilla Public License (version 1.1 or 2.0) should refer
+	  to the corresponding files under
+	  <file>/usr/share/common-licenses</file>,<footnote>
 	    <p>
 	      In particular,
               <file>/usr/share/common-licenses/Apache-2.0</file>,
@@ -10833,8 +10834,10 @@ END-INFO-DIR-ENTRY
               <file>/usr/share/common-licenses/LGPL-2</file>,
               <file>/usr/share/common-licenses/LGPL-2.1</file>,
               <file>/usr/share/common-licenses/LGPL-3</file>,
-              <file>/usr/share/common-licenses/GFDL-1.2</file>, and
-              <file>/usr/share/common-licenses/GFDL-1.3</file>
+              <file>/usr/share/common-licenses/GFDL-1.2</file>,
+              <file>/usr/share/common-licenses/GFDL-1.3</file>,
+              <file>/usr/share/common-licenses/MPL-1.1</file>, and
+              <file>/usr/share/common-licenses/MPL-2.0</file>
 	      respectively.  The University of California BSD license is
 	      also included in <package>base-files</package> as
 	      <file>/usr/share/common-licenses/BSD</file>, but given the
diff --git a/upgrading-checklist.sgml b/upgrading-checklist.sgml
index ebb17fc..8ca34a7 100644
--- a/upgrading-checklist.sgml
+++ b/upgrading-checklist.sgml
@@ -59,7 +59,11 @@ Released xxx, 2015.
   <item> recommend to ship additional documentation for package
   <prgn>pkg</prgn> in a separate package <prgn>pkg-doc</prgn> and install it
   into <file>/usr/share/doc/pkg</file>.
-  </item> </taglist></p>
+  </item>
+<tag>12.5</tag>
+  <item> Add the Mozilla Public Licenses to <file>/usr/share/common-licenses</file>;
+  so that they can be referred to rather than quoted in Copyright files.</item>
+</taglist></p>
 
 <sect id="3.9.6.0"> Version 3.9.6.0
 <p>
-- 
2.1.4

